.github/workflows/client-android-eas.yml

```
@@ -33,6 +33,7 @@ concurrency:
 jobs:
   build:
     runs-on: ubuntu-20.04
+    if: ${{ github.actor != 'dependabot[bot]' }}
```

Afaik in `if` you don't have to use interpolation
```suggestion
    if: github.actor != 'dependabot[bot]'
```